ext/search_kit/ang/crmSearchTasks/traits/searchDisplayTasksTrait.service.js

```diff
@@ -56,9 +56,10 @@
         };
         // If task uses a crmPopup form
         if (task.crmPopup) {
+          const mode = task.crmPopup.mode && task.crmPopup.mode == 'front'  ? 'front' : 'back';
```

Comment:
Perhaps we should also allow a task to specify `null` (which gets interpreted by `CRM.url` as "current mode").
```suggestion
          const mode = ('mode' in task.crmPopup) ? task.crmPopup.mode : 'back';
```